### ISSUE TITLE
Chart references a non-existent image

### ISSUE THREAD

**Issue by brsolomon-deloitte** [sighupio/gatekeeper-policy-manager#393]
```bash
$ helm pull --untar --repo https://sighupio.github.io/gatekeeper-policy-manager --version 0.2.0 gatekeeper-policy-manager

$ grep -B5 -F 'tag:' gatekeeper-policy-manager/values.yaml 

image:
  repository: quay.io/sighup/gatekeeper-policy-manager
  pullPolicy: IfNotPresent
  # Overrides the image tag whose default is the chart appVersion.
  tag: "v0.5.2-rc16"

$ docker image pull quay.io/sighup/gatekeeper-policy-manager:v0.5.2
Error response from daemon: manifest for quay.io/sighup/gatekeeper-policy-manager:v0.5.2 not found: manifest unknown: manifest unknown
$ docker image pull quay.io/sighup/gatekeeper-policy-manager:v0.5.2-rc16
Error response from daemon: manifest for quay.io/sighup/gatekeeper-policy-manager:v0.5.2-rc16 not found: manifest unknown: manifest unknown
```

**Comment by brsolomon-deloitte** [comment]
> The solution is to set the image tag that you want to use when you install the chart, see the readme for instructions:

Yep, we pinned to `image.tag = v1.0.0` (and also see that the template will use `.Chart.appVersion` as a fallback).

My two cents: 0.2.0 chart version is a bad chart release. (I do not mean to be harshly critical; just stating a fact - its own default image is missing.) It may be best to just cut a 0.3.0 chart release and set both `image.tag` and `appVersion` to v1.0.0.

**Comment by ralgozino** [comment]
> My two cents: 0.2.0 chart version is a bad chart release. (I do not mean to be harshly critical; just stating a fact - its own default image is missing.) It may be best to just cut a 0.3.0 chart release and set both `image.tag` and `appVersion` to v1.0.0.

No offence taken, I agree that as is the 0.2.0 chart is not right. But, I said previously that "I'm on the fence" about it because it doesn't solve the root problem, once we release newer versions of GPM one could say that defaulting to 1.0.0 is also wrong, for example, for security reasons: older version most likely will have CVEs.

In a weird way, the chart deployment failing when you don't set the preferred tag "is a feature and not a bug" :-) 

Pointing to `latest` is not a best practice either, but could be a middle ground solution.

**Comment by brsolomon-deloitte** [comment]
At the end of the day, the Helm chart will need to be upgraded and re-released to use the latest GPM app version as its default image version. This is what is done by, for example, [Bitnami charts](https://github.com/bitnami/charts), [fluent](https://github.com/fluent/helm-charts), and many others. 

In my opinion, a more middle-ground solution would be to pin the image tag in the helm chart to a _rolling tag_ such as `v1` or `v1.0`. (This would necessitate release those tags in addition to the full 1.0.0, as described at [Rolling Tags for Container Images](https://docs.bitnami.com/tutorials/understand-rolling-tags-containers/)). If the Helm chart pinned to `v1.0`, for instance, that would allow the downstream to receive semver patch releases but have some stability of the major and minor version release.

**Comment by ralgozino** [comment]
fixed in c02310d567eb3f8066a4871a9e13cf34fe8881fe
